exovetter/vetters.py

```diff
@@ -123,7 +129,8 @@ def run(self, tce, lightcurve):
 
         self.lpp_data = lpp.Lppdata(self.tce, self.lc, self.lc_name)
 
-        self.norm_lpp, self.raw_lpp, self.plot_data = lpp.compute_lpp_Transitmetric(self.lpp_data, self.map_info)  # noqa: E501
+        self.norm_lpp, self.raw_lpp, self.plot_data = \
+            lpp.compute_lpp_Transitmetric(self.lpp_data, self.map_info)  # noqa: E501
```

Comment:
If you want to keep within the line length limit, then you can remove `noqa` here.
```suggestion
            lpp.compute_lpp_Transitmetric(self.lpp_data, self.map_info)
```